rust/src/lib.rs

```diff
@@ -420,6 +420,12 @@ impl GraphWrapper {
         self.convert_package_dependencies_to_python(py, illegal_dependencies)
     }
 
+    pub fn find_cycles(
+        &self
+    ) -> PyResult<Vec<Vec<String>>> {
```

Comment:
🤔  To find all cycles like this sounds like it could be very computationally expensive for large graphs. Should we instead just give one start module/package to work from? e.g. `find_cycles(&self, module: &str, as_package: bool)` might be a more reasonable API.